packages/eos-components/src/utils.js

```diff
@@ -7,11 +7,15 @@ import {
 
 /** Structured tags **/
 
-export function getFirstStructuredTag(node, matchKey) {
-  if (!(matchKey in node.structuredTags)) {
-    return null;
+export function getAllStructuredTags(node, matchKey) {
+  if (!('structuredTags' in node) || !(matchKey in node.structuredTags)) {
+    return [];
```

Comment:
all of this could be done with the [`_.get`](https://underscorejs.org/#get) in just one line.

Reply:
Good. I thought this was going to error when `node.structuredTags` was undefined, but it turns `_.get()` does consider `undefined`:

```
> _.get({foo: 'bar'}.structuredTags, 'baz', [])
[]
> _.get(undefined, 'baz', [])
[]
```